bin.src/showCamera.py

```diff
@@ -25,26 +25,44 @@
 import matplotlib.pyplot as plt
 
 import lsst.afw.cameraGeom as cameraGeom
+import lsst.afw.geom as afwGeom
 import lsst.geom as geom
 
+debug = False
+
 
 def main(camera, sample=20, names=False, showDistortion=True, plot=True, outputFile=None):
+    baseFileName = outputFile.rsplit("/")[-1].split(".")[0] if outputFile else None
```

Comment:
Use `os.path.basename`?

Reply:
Yes please. Something like: ` os.path.splitext(os.path.basename(outputFile))[0]` should do it.